perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/attributestabs/CreateAttributeDefinitionTabItem.java

```diff
@@ -195,46 +182,7 @@ public void onClick(ClickEvent event) {
 					String type = typeListBox.getValue(typeListBox.getSelectedIndex());
 					boolean isUnique = unique.getValue();
 
-					CreateAttribute request = new CreateAttribute(JsonCallbackEvents.disableButtonEvents(createButton, new JsonCallbackEvents(){
-						@Override
-						public void onFinished(JavaScriptObject jso) {
-
-							AttributeDefinition a = jso.cast();
-
-							ArrayList<AttributeRights> list = new ArrayList<AttributeRights>();
-
-							AttributeRights right = AttributeRights.create(a.getId(), "SELF");
-							list.add(getRightsFromWidgets(selfRead, selfWrite, selfReadPublic, selfWritePublic, selfReadVo, selfWriteVo, right));
-
-							AttributeRights right2 = AttributeRights.create(a.getId(), "VOADMIN");
-							list.add(getRightsFromWidgets(voRead, voWrite, right2));
-
-							AttributeRights right3 = AttributeRights.create(a.getId(), "GROUPADMIN");
-							list.add(getRightsFromWidgets(groupRead, groupWrite, right3));
-
-							AttributeRights right4 = AttributeRights.create(a.getId(), "FACILITYADMIN");
-							list.add(getRightsFromWidgets(facilityRead, facilityWrite, right4));
-
-							// after update - update rights
-							SetAttributeRights request = new SetAttributeRights(JsonCallbackEvents.disableButtonEvents(createButton, new JsonCallbackEvents(){
-								@Override
-								public void onFinished(JavaScriptObject jso) {
-									enableDisableWidgets(true);
-									closeTabEvents.onFinished(jso);
-								}
-								@Override
-								public void onLoadingStart() {
-									enableDisableWidgets(false);
-								}
-								@Override
-								public void onError(PerunError error) {
-									enableDisableWidgets(true);
-								}
-							}));
-							request.setAttributeRights(list);
-
-						}
-					}));
+					CreateAttribute request = new CreateAttribute(JsonCallbackEvents.disableButtonEvents(createButton));
 					request.createAttributeDefinition(displayName, friendlyName, description, namespace, type, isUnique);
```

Comment:
before these changes the dialog window disappeared after creating new attribute definition showing success alert, but now it shows success alert and the window stays opened